src/Views/MainView.vala

```diff
@@ -18,6 +18,12 @@
 public class Sideload.MainView : AbstractView {
     public signal void install_request ();
 
+    public string app_name {
+        set {
+            primary_label.label = _("Install “%s”?").printf (value);
```

Comment:
In the issue I had something like this:

```suggestion
            primary_label.label = _("Install untrusted app “%s”?").printf (value);
```

I don't feel super strongly—it depends on if we want to keep emphasizing the "untrusted" aspect or not.  Something else to consider is that keeping "untrusted app" makes the title seem to jump around a little less, as it just loads in/adds the app name.

Something @katiemkblaede pointed out was that keeping "untrusted app" seems important to her in the case where the app you're installing has the same name as an installed trusted app—if it just said, "Install Dippi" it might be like, "yeah, but I know Dippi is trusted!" even though you could be getting a fork from a bad source.

Reply:
Oops, I did add that in there, but I guess forgot to commit it!